upgrade_test.py

```diff
@@ -456,36 +456,36 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
             # write workload during entire test
```

Comment:
Remove the two lines above:
self.prepare_keyspaces_and_tables()
            self.fill_and_verify_db_data('BEFORE UPGRADE', pre_fill=True)	            self.fill_and_verify_db_data('BEFORE UPGRADE', pre_fill=True)